mappings/net/minecraft/client/gui/ChatPreview.mapping

```diff
@@ -0,0 +1,22 @@
+CLASS net/minecraft/unmapped/C_fijiyucq net/minecraft/client/gui/ChatPreview
+	FIELD f_bflusjjm handler Lnet/minecraft/unmapped/C_jhskzigo;
+	FIELD f_bydpkiqb responce Lnet/minecraft/unmapped/C_fijiyucq$C_hzhktnuf;
+	FIELD f_meljmqvn currentMessage Ljava/lang/String;
+	FIELD f_yownppzk previousMessage Ljava/lang/String;
+	METHOD <init> (Lnet/minecraft/unmapped/C_ayfeobid;)V
+		ARG 1 client
+	METHOD m_entxrgme update (Ljava/lang/String;)V
+		ARG 1 text
+	METHOD m_flcbreqi tick ()V
+	METHOD m_huzdzaoj trimString (Ljava/lang/String;)Ljava/lang/String;
+		ARG 0 str
```

Comment:
```suggestion
		ARG 0 string
```

Reply:
why is `string` preferred over `str`?

Reply:
`str` is not one of [the abbreviations we use](https://github.com/QuiltMC/quilt-mappings/blob/1.19.1-pre5/CONVENTIONS.md#abbreviations), nor is that even a concept in Java itself, we don't have primitive strings like other languages do